run-tests-with-docker-parallel.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+set -u
```

Comment:
Do you not want `-e` in here as well?

Reply:
🤷‍♂️ I just duplicated the existing `run-tests-with-docker.sh` script